Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -207,6 +212,66 @@ Describe "New-GcsObject" {
         $obj.Metadata["Content-Type"] | Should Be "image/jpeg"
         Remove-GcsObject $obj
     }
+
+    It "should upload an empty folder" {
+        $TestFolder = Join-Path $TestDrive "TestFolder"
```

Comment:
Also, where is `$TestDrive` defined? Why don't we just use `GetRandomFileName` [link]((https://msdn.microsoft.com/en-us/library/system.io.path.getrandomfilename(v=vs.110).aspx) instead? That would obviate the need for the if-statement later.